app/controllers/api/v1/flavors_controller.rb

```diff
@@ -0,0 +1,14 @@
+class Api::V1::FlavorsController < ApplicationController
+  def index
+  end
+  
+  def show
+    # binding.pry
+    @region = Region.find(params[:region_id])
+    @flavor = Flavor.find(params[:id])
+    render json: @flavor
+    # @flavor = Region.find(params[:id])
+    # @flavors = @region.flavors
+    # render json: @flavors.order('name')
+  end
```

Comment:
- remove all commented out code. 
- if the `index` action is not being used in this PR, it shouldnt be presen
- we arent doing anything with @region here. I can see some value to it, but it isnt being used in this PR. Specifically, maybe on the flavor show page I can see all regions that this flavor is a part of?

Reply:
Deleted commented out code